x-pack/test_serverless/api_integration/test_suites/common/search_oss/index.ts

```diff
@@ -16,6 +16,5 @@ export default function ({ loadTestFile }: FtrProviderContext) {
     loadTestFile(require.resolve('./search'));
     // TODO: Removed `sql_search` since
     // SQL is not supported in Serverless
-    loadTestFile(require.resolve('./bsearch'));
```

Comment:
I think it still makes sense to keep `bsearch` tests as long as we have `/internal/bsearch` endpoint.

Reply:
It's still being called places after this PR merges, right? If so I agree, it would probably be best to remove the tests when `/internal/bsearch` is removed.